corehq/apps/hqcase/tasks.py

```diff
@@ -22,11 +23,13 @@ def explode_cases(user_id, domain, factor, task=None):
     old_to_new = dict()
     child_cases = list()
 
+    case_ids = get_case_ids_in_domain_by_owner(
```

Comment:
i know you haven't included this on any of your interfaces (that i've seen), but it may be nice to allow the user to pass a flag for `include_docs`. we avoid a second call to couch then. understand it might be a bit confusing to have `get_case_ids.*` return cases. thoughts?